Decred Wallet/Features/Send/SendViewController.swift

```diff
@@ -423,13 +425,13 @@ class SendViewController: UIViewController {
                     progressHud.dismiss()
                     
                     if error.localizedDescription != DcrlibwalletErrInvalidPassphrase {
-                        self.showOkAlert(message: error.localizedDescription, title: "Error")
+                        self.showOkAlert(message: error.localizedDescription, title: LocalizedStrings.error)
                         return
                     }
                     
                     let securityType = SpendingPinOrPassword.currentSecurityType()!.lowercased()
-                    let errorMessage = "You entered an incorrect \(securityType). Try again?"
-                    self.showOkAlert(message: errorMessage, title: "Failed to send transaction", okText: "Retry", onPressOk: self.attemptSend, addCancelAction: true)
+                    let errorMessage = String(format: LocalizedStrings.incorrectSecurityType, securityType)
```

Comment:
`incorrectPassphrase` or `incorrectSecurityInfo`, it's definitely not the security type that was incorrect.